arbitrum-docs/partials/_troubleshooting-nodes-partial.md

```diff
@@ -24,11 +24,11 @@
 
 ### How do I read messages from the Sequencer feed? {#how-do-i-read-messages-from-the-sequencer-feed}
 
-<p>An Arbitrum full node run as an <a href="https://developer.offchainlabs.com/node-running/running-a-node#arb-relay">Arb-Relay </a>can subscribe to the Sequencer feed at for real-time data as the Sequencer accepts and orders transactions off-chain. </p>
+<p>Running an Arbitrum relay locally <a href="https://developer.offchainlabs.com/node-running/running-a-node#arb-relay">Arb-Relay </a> gives users the ability to subscribe to the Sequencer feed at for real-time data as the Sequencer accepts and orders transactions off-chain. </p>
```

Comment:
Since you changed the title of the header on `running-a-node.mdx` from Arb-Relay to Feed Relay, the link text here should also be changed to Feed Relay.

Reply:
And the hash `#arb-relay` should also be changed

Reply:
updated link.

The classic relay was actually called `arb-relay`, but changed to reduce confusion.